README.md

```diff
@@ -904,19 +904,19 @@ create operations so the full context will return twelve products.
 Context data for a specified entity type can be retrieved by making a GET
 request to the `/v2/entities` endpoint and supplying the `type` parameter.
 Combine this with `options=count` and `attrs=id` to return the `id` attributes
-of the given `type`
+of the given `type`.
 
 ## Update Operations
 
 Overwrite operations are mapped to HTTP PUT. HTTP PATCH can be used to update
 several attributes at once.
 
--   The `/v2/entities/<entity>/attrs/<attribute>/value` endpoint is used to
+-   The `/v2/entities/<entity>/attrs/<attribute>/value` endpoint can be used to
     update an attribute
--   The `/v2/entities/<entity>/attrs` endpoint is used to update multiple
+-   The `/v2/entities/<entity>/attrs` endpoint can be used to update multiple
     attributes
 
-### Overwrite the value of an Attribute value
+### Overwrite an Attribute's value
```

Comment:
**the value of an Attribute** is more formal and use of an apostrophe in a title looks sloppy.

Please revert.

Reply:
done, thanks!!!